packages/plugin-infra/src/web3-token-types.ts

```diff
@@ -0,0 +1,87 @@
+export enum EthereumTokenType {
```

Comment:
Here is not a good place for EVM token types. Why not move those types to `'@masknet/web3-shared-evm'`?

Reply:
I'm supposed to rename it to `NonFunfibleTokenType` and move them to `@masknet/web3-shared-base`

Reply:
At `@masknet/plugin-infra`, I created an `useNonFungibleAssets()` which uses `ERC721TokenDetailed`. `@masknet/plugin-infra` can not import things from `'@masknet/web3-shared-evm'` since it would cause a cycle.